gui/sharing/models.py

```diff
@@ -127,6 +133,16 @@ class CIFS_Share(Model):
         blank=True,
         null=True
     )
+    cifs_vuid = models.CharField(
+        max_length=36,
+        verbose_name=_('vuid for Time Machine'),
+        blank=True,
+        help_text=_(
+            'Volume UUID that will be advertized for time machine. '
+            'This value will be automatically generated when the share '
+            'is configured for time machine.'
+        ),
+    )
```

Comment:
Do we need to expose this to the user? I would think not?
editable=False should be added here if thats the case.